### PR TITLE
Katas UI review - Multiqubit measurements

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**

### CODE REVIEW

File: katas/content/multi_qubit_measurements/index.md [hunk 1]
```diff
@@ -155,14 +157,14 @@ After this, the probabilities of measuring each of the four basis vectors is giv
...
 
-1. We start by preparing the state $\ket \psi$.
-   To do this, we can represent $\ket \psi$ as follows:
+1. You start by preparing the state $\ket \psi$.
+   To do this, you can represent $\ket \psi$ as follows:
    $$\frac 2 3 \ket{00} + \big( \frac 1 {\sqrt 5} \ket{0} + \frac 2 {\sqrt 5} \ket{1} \big) \frac {\sqrt 5} 3 \ket{1}$$
-   This representation tells us how we should rotate individual qubits.
-2. To figure out the measurement outcome probabilities in the computational basis, we can just use the `DumpMachine` function that lists probabilities associated with each basis state present in the superposition.
-3. To figure out the measurement outcome probabilities in the Pauli X basis, we can apply a transformation that maps the two-qubit Pauli X basis into the two-qubit computational basis. This transformation just applies a Hadamard gate to each of the qubits.
+   This representation tells you how we should rotate individual qubits.
```
Comment (tcNickolas) [inline katas/content/multi_qubit_measurements/index.md:165]: There's a leftover we

File: katas/content/multi_qubit_measurements/state_preparation/solution.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+* If the result is $\ket{1}$, the first two qubits collapse to a state $\ket{11}$, so your goal is not achieved. The good thing is, this only happens in 25% of the cases, and you can just reset our qubits to the $\ket{00}$ state and try again.
 
 > Q# has a built-in <a href="https://learn.microsoft.com/azure/quantum/user-guide/language/expressions/conditionalloops#repeat-expression" target="_blank">repeat-until-success (RUS) loop</a>, which comes in handy in this case.
-> * We will describe the main operations (applying $H$ and $CCNOT$ gates and the measurement) in the `repeat` part of the loop, which specifies its body.  
-> * `until` section specifies the condition which will break the loop. In this case the result of the measurement needs to be `Zero` to indicate our success.  
-> * Finally, the `fixup` section allows us to clean up the results of the loop body execution before trying again if the success criteria is not met. In this case we reset the first two qubits back to the $\ket{00}$ state.
+> * You'll describe the main operations (applying $H$ and $CCNOT$ gates and the measurement) in the `repeat` part of the loop, which specifies its body.
```
Comment (tcNickolas) [inline katas/content/multi_qubit_measurements/state_preparation/solution.md:21]:
```suggestion
> * The main operations (applying $H$ and $CCNOT$ gates and the measurement) are described in the `repeat` part of the loop, which specifies its body.  
```

**Review by tcNickolas — APPROVED** [review]
Looks good overall, just two small comments

File: katas/content/multi_qubit_measurements/index.md (hunk 1 above)
Comment (SoniaLopezBravo) [reply katas/content/multi_qubit_measurements/index.md:165]:
```suggestion
   This representation tells you how you should rotate individual qubits.
```

File: katas/content/multi_qubit_measurements/index.md [hunk 3]
```diff
@@ -243,21 +245,21 @@ Full measurements can also be used to identify the state of the system, if it is
     "title": "Partial Measurements"
 })
 
-For a system with $n>1$ qubits, it is possible to measure $m<n$ qubits one after another. The number of measurement outcomes is then $2^m$ instead of $2^n$. The probabilities of each of the outcomes and the post-measurement states of the qubits can be found using the projection formalism for measurements.
+For a system with $n>1$ qubits, it's possible to measure $m<n$ qubits one after another. The number of measurement outcomes is then $2^m$ instead of $2^n$. The probabilities of each of the outcomes and the post-measurement states of the qubits can be found using the projection formalism for measurements.
 
-First, we recall the concept of projection operators introduced in the Measurements in Single-Qubit Systems kata. Measurements are modeled by orthogonal projection operators - matrices that satisfy
+First, you recall the concept of projection operators introduced in the Measurements in Single-Qubit Systems kata. Measurements are modeled by orthogonal projection operators - matrices that satisfy
```
Comment (cesarzc) [inline katas/content/multi_qubit_measurements/index.md:250]:
```suggestion
First, recall the concept of projection operators introduced in the Measurements in Single-Qubit Systems kata. Measurements are modeled by orthogonal projection operators - matrices that satisfy
```

Comment (cesarzc) [reply katas/content/multi_qubit_measurements/index.md:250]: I think in this case it's better to remove "you" from the sentence.

**Review by cesarzc — APPROVED** [review]
Just left one small suggestion.

**Pull request merged**